src/integration-test/java/com/gradle/exportapi/DatabaseBaseIntegrationTest.java

```diff
@@ -0,0 +1,28 @@
+package com.gradle.exportapi;
+
+import org.junit.After;
+import org.junit.Before;
+import org.knowm.yank.PropertiesUtils;
+import org.knowm.yank.Yank;
+
+import java.util.Properties;
+
+public abstract class DatabaseBaseIntegrationTest {
+
+    @Before
+    public void setup() {
+        Yank.setThrowWrappedExceptions(true);
+        Properties dbProps = PropertiesUtils.getPropertiesFromClasspath("POSTGRES.properties");
```

Comment:
So tests run against the same production database ge-export normally points at?
I don't want executing tests to alter the production db.

Ideally we could setup a lightweight, file-based database like sqlite (assuming there are no meaningful incompatibilites with postgresql). Failing that, tests should at least default to a different database than the production database.